helm/gcp-compute-persistent-disk-csi-driver/templates/v1_serviceaccount_csi-gce-pd-controller-sa.yaml

```diff
@@ -3,6 +3,8 @@ kind: ServiceAccount
 metadata:
   annotations:
     giantswarm.io/gcp-workload-identity-pool-id: '{{.Values.gcpProject}}.svc.id.goog'
```

Comment:
I think we should change this line, because I think it's better than the whole string is passed rather than making the app construct `{{.Values.gcpProject}}.svc.id.goog`.

Reply:
Same for the identity provider below.

Reply:
I agree. I'll change the identity provider annotation to do the same